apps/users/models.py

```diff
@@ -372,6 +385,9 @@ def generate_confirmationcode(self):
                                                           string.digits, 60))
         return self.confirmationcode
 
+    def set_unusable_password(self):
+        self.password = ''
```

Comment:
That's what this will use otherwise but we overwrite the`User.has_usable_password` and `User.check_password` so that doesn't work.